src/routes/post.jsx

```
@@ -57,20 +93,22 @@ export default function Post() {
                 bg-[#4B7EFF]
                 text-white
                 font-bold
                 rounded-[5px]
                 py-[2px]
                 px-[5px]
                 "
-              >
-                #공동구매
-              </button>
+                >
+                  #{tag}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)